Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/FontDesignModifier.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  FontDesignModifier.swift
+// LiveViewNative
+//
+//  Created by May Matyi on 4/24/23.
+//
+
+import SwiftUI
+
+/// Applies a font design to any element.
+///
+/// <Text modifiers={font_design(@native, design: :default)}>This text is default</Text>
+/// <Text modifiers={font_design(@native, design: :monospaced)}>This text is monospaced</Text>
+/// <Text modifiers={font_design(@native, design: :rounded)}>This text is rounded</Text>
+/// <Text modifiers={font_design(@native, design: :serif)}>This text is serif</Text>
```

Comment:
This should be in a codeblock or it won't render.

Reply:
Oops, good catch